README.adoc

```diff
@@ -0,0 +1,205 @@
++++
+<img align="right" role="right" width="180" src="/docs/img/openshift.png?raw=true"/>
```

Comment:
Hey @Preeticp I've looked at your doc and I believe you did too big of a compression to the image... It's really really blurry? Perhaps keep the original file and just re-size. 

No idea what happened, but looks like modified the image and now it's super pixelated.

Reply:
See the file here: https://github.com/openshift/odo/blob/934b5412fc8a5f6aec1cea89c166fc3a91c0b0f7/README.adoc